packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1649,6 +1726,11 @@ export class ZWaveController
 			strategy: InclusionStrategy.SmartStart,
 			provisioning: provisioningEntry,
 		};
+		this._inclusionFlags = {
+			highPower: true,
+			networkWide: true,
+			protocolLongRange: provisioningEntry.isLongRange,
```

Comment:
I didn't understand all the inclusion pathways well enough to know what the right thing to do here is. There's a separate smartstart message for LR nodes (Update Smart Start HomeID (LR) = 0x87). Again, it wasn't clear if that's a statement about "I'm long range capable" or "I must be included long range". If the former, what is here is correct. If the latter, we'll want to allow the user to decide _somwhere_ whether this node is being added as long range or normal. At the time they add/verify the DSK, there should probably be a "as long range if possible" checkbox?